website/content/docs/concepts/filtering/audit.mdx

```diff
@@ -0,0 +1,157 @@
+---
+layout: docs
+page_title: Audit Filtering
+description: |-
+  How to filter audit entries in Vault.
+---
+
+[filter syntax]: /vault/docs/concepts/filtering
+
+# Filtering audit entries
+
+This page describes how to use filters with Vault audit devices to fine tune which
+audit entries should be written to an audit log.
+
+Starting in Vault 1.16.0, audit devices can be enabled with a `filter` option, which
+is used to evaluate audit entries and decide if they should be written to a particular
+audit log. Please ensure you are familiar with the [filtering concept](/vault/docs/concepts/filtering) before
+attempting to configure an audit device with a filter.
+
+<Note title="Advanced audit feature">
+  The use of filtering in Vault's audit system should be considered an advanced feature.
+  Whilst it has been designed to be simple and flexible, exclusively enabling filtered
+  devices could result in some requests and responses that are not audited.
+</Note>
+
+Please see the [Vault security model](/vault/docs/internals/security) for further information.
+
+## `filter` option
+
+All audit device types ([file](/vault/docs/audit/file), [socket](/vault/docs/audit/socket)
+and [syslog](/vault/docs/audit/syslog)) support the `filter` option at the time they
+are enabled. After successfully enabling a device with a filter, every audit entry
+that Vault sends to that audit device will be compared to the expression in the filter.
+Only audit entries that match the filter should then be written to the device's audit log.
+
+### Valid `filter` properties
+
+Filters can reference the following properties of an audit entry:
+
+* `mount_point` - path to the mount (e.g. [auth method](/vault/docs/auth), [secret engine](/vault/docs/secrets)),
```

Comment:
This is a stylistic comment that I'm not 100% confident in, but I think we usually start bullet points off with a capital? I'm not sure which is 'right' though, so I'll definitely defer to the docs folks on this one, but I wanted to raise it at the very least